requirements.txt

```
@@ -2,18 +2,19 @@ alembic>=1.3.1, <2.0.0
 arrow>=0.14.6, <2.0.0
 defusedxml>=0.6.0, <1.0.0
 fedora_messaging>=2.0.1, <3.0.0
-flask>=1.1.4, <3.0.0
 flask-login>=0.4.1, <1.0.0
 flask-wtf>=0.14.2, <1.0.0
+flask>=1.1.4, <3.0.0
 jinja2>=2.11.2, <4.0.0
 ordered-set>=3.1, <5.0.0
 python-dateutil>=2.8.0, <3.0.0
-pytoml>=0.1.18, <1.0.0
 semver>=2.8.1, <3.0.0
-social-auth-app-flask>=1.0.0, <2.0.0
 social-auth-app-flask-sqlalchemy>=1.0.1, <2.0.0
+social-auth-app-flask>=1.0.0, <2.0.0
+sqlalchemy>=1.3.17, <2.0.0
 sseclient>=0.0.26, <1.0.0
 straight.plugin>=1.5.0, <2.0.0
-sqlalchemy>=1.3.17, <2.0.0
+toml==0.10.2
```

Don't use exact versions, this could fail if version is not available in Fedora yet
```suggestion
toml>=0.10.2, <1.0.0
```